components/PrimaryLayout.js

```diff
@@ -27,3 +28,7 @@ export default function PrimaryLayout({ children }) {
     </>
   );
 }
+
+PrimaryLayout.propTypes = {
+  children: PropTypes.node,
```

Comment:
maybe we could have this `isRequired`?

Reply:
Left as not required as there isn't always a card available for some sections, particularly if they're in-dev, for these sections we still need to display the rest of the layout.